config/api.go

```diff
@@ -93,15 +75,12 @@ func (a API) Validate() error {
 // DefaultAPI returns the default configuration details
 func DefaultAPI() *API {
 	return &API{
-		Enabled:          true,
-		Address:          DefaultAPIAddress,
-		WebsocketAddress: DefaultAPIAddress,
+		Enabled: true,
+		Address: DefaultAPIAddress,
 		AllowedOrigins: []string{
-			"electron://local.qri.io",
-			"http://app.qri.io",
-			"https://app.qri.io",
+			fmt.Sprintf("http://localhost:%s", DefaultAPIPort),
 		},
-		DisableWebui: false,
+		EnableWebui: true,
```

Comment:
Unsure about the default state of this. The rename to me implies that the default is `false` and then you enable it by config. I'm fine with it to stay as is, but think `DisableWebui` reflected the basic assumptions better.

Reply:
I am ok with the rename to `EnableWebui`. I think generally, defaults for bool fields should be false, but we already break that by having `Enabled: true` field up a little bit higher. With that in mind, at least getting rid of the double negative is being more consistent.

Reply:
Yeah I'm generally with @dustmop's view on this one. How y'all feel about just `Webui` instead of `EnableWebui`?
`Webui: true` feels like "would you like webui? yes"

Reply:
Yeah, I'm ok with `Webui` as well.

Reply:
+1